onnxruntime_extensions/pnp/_nlp.py

```diff
@@ -21,8 +21,8 @@ def create_bert_tokenizer(ctx, name, input_names, output_names, container, opera
         attrs = dict(vocab_file=vocab)
         # Unfortunately, there's no specific accessor function on
         # transformers.BertTokenizer to query for strip_accents.
-        attrs['strip_accents'] = 1 if hf_bert_tokenizer.init_kwargs.get('strip_accents') else 0
-        attrs['do_lower_case'] = 1 if hf_bert_tokenizer.do_lower_case else 0
+        attrs['strip_accents'] = 1 if hasattr(hf_bert_tokenizer, 'do_lower_case') and hf_bert_tokenizer.init_kwargs.get('strip_accents') else 0
```

Comment:
should it be 'strip_accents'?

Reply:
Yes, sorry, my mistake good catch. Fixed :)